sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/ControlFlowGraph/ControlFlowGraphTest.cs

```diff
@@ -2669,7 +2715,68 @@ public void Cfg_TryCatch_ThrowInsideTry()
 
         [TestMethod]
         [TestCategory("CFG")]
-        public void Cfg_TryCatch_Inside_DoWhile()
+        public void Cfg_TryCatch_WithBreak_Inside_DoWhile()
+        {
+            var cfg = Build(@"
+            var attempts = 0;
+            do
+            {
+                cw0();
+                try
+                {
+                    attempts++;
+                    cw1();
+                    break;
```

Comment:
Is there a test where there is a `break` inside a `try-catch-finally` (or `try-finally`, and `try-catch-catch-finally`)?

Reply:
in addition, I'll have to add tests with a `continue` inside the `try-finally`